### DISCUSSION TITLE
Feature Request: Overlay Collections Across Scenes

### DISCUSSION THREAD

**Discussion by Sci-fer** [UPBGE/upbge#1703] [Ideas/Features for UPBGE 0.3+]
Up until version 0.2, GUIs where made with Scene actuator in Logic Bricks. Now in version 0.3, this functionality was moved to Collection actuator.

The problem is we cannot put the overlay content in **another scene** anymore.

Just try moving HUD collection to another scene, in the following project, to see it stops working.
[HUD-test.blend](https://drive.google.com/file/d/1d_viEO8NYqgriZEnliQjJrdS1yaf4Tkc/view?usp=sharing)

Please restore this feature.

**Comment by BluePrintRandom** [comment]
We have overlay collection now,

The concept of multiple scene rendering, is replaced by multiple collections. 

```
for ob in overlay_collection.all_objects:
   bpy.data.scenes['SceneName'].collection.objects.link(ob)
   
```
I think something like this is what you need to have it run in the other scene*

**Reply by SwanTheHunt** [reply]
Now, the mouse overlay not working :/

**Comment by BluePrintRandom** [comment]
edited**

**Comment by Sci-fer** [comment]
I thought that was a bug.

Anyway, I hate Logic Bricks.

Thanks for the suggestion!
